### DISCUSSION TITLE
Is it possible to prefer a language for extras?

### DISCUSSION THREAD

**Discussion by saschabrockel** [RandomNinjaAtk/docker-radarr-extended#40] [Q&A]
Hey there. I used another trailer-downloader before but this one was not completely automated. Now I am wondering if it would be possible to say like I did I want to have extras for German and English (`extrasLanguages=de-DE,en-US`) that English is only downloaded if German is not available. Because I don't need English if I got it in German. And it's not nice because Plex picks a random one if you just click the button play trailer (and often it is the English one).

**Comment by RandomNinjaAtk** [comment]
It's possible to grab multiple languages, but not capable of of having a preferred lang with the other language as a fallback. If you set it to single trailer only, then it may work in the manor your requesting, but it's untested.

**Reply by saschabrockel** [reply]
Thank you for the answer. Would it be possible to add a feature like this?

**Reply by RandomNinjaAtk** [reply]
It is likely possible in some shape/form but I personally don't have a use for it. I'll gladly review a PR for an implementation.
